src/screens/ChannelListScreenV2/index.tsx

```diff
@@ -26,6 +31,82 @@ const ChannelListScreenV2 = () => {
     navigation.navigate('ContactScreen');
   };
 
+  const requestPermission = async () => {
+    const authStatus = await messaging().requestPermission();
+    const enabled =
+      authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
+      authStatus === messaging.AuthorizationStatus.PROVISIONAL;
+
+    if (enabled) {
+      const fcmToken = await messaging().getToken();
+      const payload = {
+        fcm_token: fcmToken
+      };
+      fcmTokenService(payload);
+    }
+  };
+
+  React.useEffect(() => {
+    const checkNotif = async () => {
+      if (!navigation.isFocused()) return;
+      PushNotification.checkPermissions(
+        (
+          data: PushNotificationPermissions & {
+            authorizationStatus?: number;
+          }
+        ) => {
+          const lastPromptTime = StorageUtils.lastPromptNotification.get();
+
+          // For testing purpose,  (20000 milliseconds)
+          const promptInterval = 48 * 3600000;
```

Comment:
please change to 
`48 * 60 * 60 * 1000` for better readability

Reply:
ah okay